akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -754,7 +754,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
 
   def receiveGossipStatus(status: GossipStatus): Unit = {
     val from = status.from
-    if (!latestGossip.overview.reachability.isReachable(selfUniqueAddress, from))
+    if (!latestGossip.isReachable(selfUniqueAddress, from))
```

Comment:
No, I thought it should be the other way around, that if there is inter-team unreachability, we shouldn't care about that when getting gossip from inside our team. I think without this, the gossip will not be seen by all team members in case of inter-team unreachability and hinder leader actions.